plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/Messages.java

```diff
@@ -65,6 +65,7 @@ public final class Messages extends NLS {
 	public static String Error_RecursiveType;
 	public static String Error_SelfInsertion;
 	public static String ErrorMarkerInterfaceAnnotations_MissingVariableForAttribute;
+	public static String ErrorMarkerInterfaceAnnotations_MissingAttribute;
```

Comment:
Please rename this text:
```suggestion
	public static String AttributeAnnotations_MissingAttributeDeclaration;
```